website_sale_product_eprel/views/templates.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+  <template
+        id="product_eprel_label"
+        inherit_id="website_sale.product"
+        name="EPREL: Energy Label in Product Page"
```

Comment:
You can add the customize_show attribute to the view and enable it to be activated/deactivated from the web e